### PR TITLE
Akka HTTP -> ADOPT

### PR DESCRIPTION
**Pull request by umatrangolo**

### CODE REVIEW

File: recommendations/frameworks/Scala.md [hunk 1]
```diff
@@ -5,6 +5,13 @@
 
   - [Play](https://www.playframework.com/)
 
+  - [Akka HTTP](http://doc.akka.io/docs/akka/2.4.11/scala/http/)
+    We think that for writing lean microservices wout a web frontend
```
Comment (gheine) [inline recommendations/frameworks/Scala.md]: "without"

Comment (umatrangolo) [inline recommendations/frameworks/Scala.md]: Done.

**Comment by umatrangolo** [comment]
I feel pretty strong with it. I'm really enjoying its DSL to write routes and its tight, opinionated approach to streams and actors. 

I call for a vote here :)

**Comment by jkenny** [comment]
@gheine aren't you trialling a different one on mobile?

**Comment by gheine** [comment]
@jkenny yes, but the jury is still out if that will become a recommendation

**Comment by arbjerg** [comment]
Voting on `asses` vs `trial` - asses won!

**Comment by arbjerg** [comment]
Voted to Asses!

**Comment by gheine** [comment]
@umatrangolo mind updating the PR (trial -> assess) and merging afterwards?

**Comment by umatrangolo** [comment]
👍

**Pull request merged**